Dockerfile

```diff
@@ -16,47 +16,50 @@
 
 FROM ubuntu:14.04
```

Comment:
Please reset this Dockerfile to the version from develop and only re-apply necessary changes.

Reply:
This is also done.